integrations/micrometer/micrometer/src/main/java/module-info.java

```diff
@@ -32,17 +32,17 @@
 
     requires java.logging;
 
-    requires static jakarta.annotation;
+    requires jakarta.annotation;
 
     requires io.helidon.config;
-    requires io.helidon.common.http;
-    requires io.helidon.nima.servicecommon;
+    requires transitive io.helidon.common.http;
+    requires transitive io.helidon.nima.servicecommon;
     requires io.helidon.nima.webserver.cors;
 
     requires static io.helidon.config.metadata;
 
-    requires micrometer.core;
-    requires micrometer.registry.prometheus;
+    requires transitive micrometer.core;
+    requires transitive micrometer.registry.prometheus;
```

Comment:
Prometheus registry is not part of the public API of this module, so it should not be transitive

Reply:
ok